yb-voyager/cmd/exportData.go

```diff
@@ -90,6 +90,9 @@ func exportDataCommandFn(cmd *cobra.Command, args []string) {
 	if useDebezium && !changeStreamingIsEnabled(exportType) {
 		utils.PrintAndLog("Note: Beta feature to accelerate data export is enabled by setting BETA_FAST_DATA_EXPORT environment variable")
 	}
+	if changeStreamingIsEnabled(exportType) {
+		utils.PrintAndLog(color.YellowString(`Note: Live migration feature [TECH PREVIEW] is enabled by setting --export-type "%s"`, exportType))
```

Comment:
The wording somehow feels ambiguous. (It could be interpreted as  - you CAN enable it by setting --export-type ...)

How about 
`[TECH PREVIEW] live migration feature has been enabled as --export-type %s is set `

Reply:
`Note: Live migration is a TECH PREVIEW feature. Not meant for production use.`